src/config.rs

```diff
@@ -113,6 +111,25 @@ mod arg {
   pub(crate) const DUMP_FORMAT_VALUES: &[&str] = &[DUMP_FORMAT_JUST, DUMP_FORMAT_JSON];
 }
 
+mod arg_or_cmd {
```

Comment:
Not too keen on the naming here, but I figured this should be different to `arg` as it serves 2 purposes now.

Reply:
On reflection and looking at the code, I think we should avoid overloading the `--evaluate` command flag with a second meaning. It complicates the argument parser and makes the help text weird. How about `--list --eval`?